flatbuffer-schemes/input-events.fbs

```diff
@@ -3,10 +3,26 @@
 namespace CoffeeTime.InputEvents;
 
 // Produced by the Screen when a player interacts with the keyboard
-table KeyboardEvent {
-    state:string; // could be released or pressed
-    name:string; // keyboard keys
+table Keyboard {
+    type:string; // could be released or pressed
+    key:string; // keyboard keys
 }
 
-root_type KeyboardEvent;
+table Position {
+    x:float;
+    y:float;
+}
+
+table Mouse {
+    type:string;
+    button:string;
+    position:Position;
```

Comment:
You don't want to receive the motion events, right?

In other words, from the screen, only the pressed and released events are sent with the last position of the mouse?

Reply:
Yes, I think that will suffice for now